src/safemode_ui.cpp

```diff
@@ -599,7 +599,7 @@ rule_state safemode::check_monster( const std::string &creature_name_in,
     if( iter != safemode_rules.end() ) {
         const auto &tmp = ( iter->second )[( int )attitude_in];
         if( tmp.state == RULE_BLACKLISTED ) {
-            const int check_dist = ( tmp.proximity == 0 ) ? 50 : tmp.proximity;
+            const int check_dist = ( tmp.proximity == 0 ) ? 60 : tmp.proximity;
```

Comment:
This check is not even required but I need a value for the SAFEMODEPROXIMITY option which is at max 50 as well atm.
Can't seem to find out how this is actually calculated though.